app/app/urls.py

```diff
@@ -115,8 +116,15 @@
     path('actions/tool/<int:tool_id>/voteDown', dashboard.views.vote_tool_down, name='vote_tool_down'),
     url(r'^tools/?', dashboard.views.toolbox, name='tools'),
     url(r'^gas/?', dashboard.views.gas, name='gas'),
+
+    # redeem coin
     url(r'^coin/redeem/(.*)/?', dashboard.views.redeem_coin, name='redeem'),
 
+    # EthOS
+    url(r'^ethos/proxy/?', ethos.views.proxy, name='proxy_image'),
```

Comment:
Can we move this to `app/ethos/urls.py` and `include` the ethos `namespace` here?

So your ethos/urls.py would look like:

```python
from django.urls import re_path

from . import views

app_name = 'avatar'
urlpatterns = [
    re_path(r'^proxy/?', views.proxy, name='proxy_image'),
    re_path(r'^tweet/?', views.tweet_to_twitter, name='tweet_to_twitter'),
    re_path(r'^(.*)/?', views.redeem_coin, name='redeem_coin'),
]
```

and app/urls.py:

```python
    path('ethos/', include('ethos.urls', namespace='ethos')),
```